apps/demo-nextjs/package.json

```diff
@@ -18,6 +18,6 @@
     "next": "13.4.4",
     "react-dom": "18.2.0",
     "react": "18.2.0",
-    "typescript": "^4.8.4"
+    "typescript": "^5.1.0"
```

Comment:
The build was failing because I've removed also removed the TS version from `providers/package.json` and thus defaulted it to the repo version which is ~5.1.6.